website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+:::
+
+For more details on this change, including migration steps and how this affects Flow users, check out our previous post [_First-class Support for TypeScript_](https://reactnative.dev/blog/2023/01/03/typescript-first).
+
+## Simplifying layouts with Flexbox gap
+
+With React Native you can flexibly layout components on different screen sizes using flexbox. Browsers have supported the flexbox properties [_gap_](https://developer.mozilla.org/en-US/docs/Web/CSS/gap), [_rowGap_](https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap), and [_columnGap_](https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap), which allow you to specify the amount of space between all items in a flexbox.
+
+These properties have been long requested in React Native, and 0.71 adds initial support for gaps defined using pixel values. In future version, we will add support for more values, such as percentages.
+
+To see why this is useful, imagine trying to build a responsive layout with variably sized cards, 10px apart from each other, hugging the edges of the parent container. Trying to accomplish this layout with child margins can be tricky.
+
+The following shows a layout where we start by giving each child `margin: 10`:
+
+![Two diagrams. On the left it shows a skeleton of an app with three boxes that have margin around them cause the boxes to have margin around all sides. On the right, the same diagram is shown highlighted to demonstrate the margin on all sides.](/blog/assets/FlexboxGapBefore.png)
+
+Margins are applied uniformly to the edges of all children and don’t collapse under flexbox, giving us spacing at the exterior of the cards, and double the space on the interior compared to what we wanted. We can get around this by applying non-uniform margins, using negative margins on the parent, halving our intended spacing, etc, but it can be made much easier.
+
+With flex gap, this layout can be achieved by setting `gap: 10` for a 10px gap between the interior of the cards:
+
+![[Two diagrams. On the left it shows a skeleton of an app with three boxes that have margin only on the inner sides and not the outer sides of the boxes due to the Flexbox gap property. On the right, the same diagram is shown highlighted to demonstrate the margin only on the inner sides.](/blog/assets/FlexboxGapAfter.png)
+
+For more information on Flexbox gaps, see [_https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps_](https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps).
+
+## Web-inspired props for accessibility, styles, and events
+
+:::note
+
+For more background on aligning React Native to web standards, check out this [_proposal_](https://github.com/necolas/discussions-and-proposals/blob/reduce-fragmentation/proposals/0000-reduce-fragmentation.md) and [_related discussion_](https://github.com/react-native-community/discussions-and-proposals/pull/496).
```

Comment:
The other sections that have a note have it at the bottom while this one has it at the top. Should it be at the bottom so we don't send people to another link before the read what's here?